O/ORTools/build_tarballs.jl

```diff
@@ -0,0 +1,120 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "ORTools"
+version = v"9.7"
+
+# Collection of sources required to build this package
+sources = [
+    GitSource("https://github.com/google/or-tools.git",
+              "6fa02e157a5c91067b7d7b88629472b9ed461193"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+# Prepare the source directory.
+cd $WORKSPACE/srcdir/or-tools*
+atomic_patch -p1 "${WORKSPACE}/srcdir/patches/cmake_dependencies_CMakeLists.txt.patch"
+mkdir build
+cmake --version
+
+ls -R /cache/julia-buildkite-plugin
+
+if [[ "$MACHTYPE" == *musl ]]
+then
+  curl -o julia-1.9.3.tar.gz https://julialang-s3.julialang.org/bin/musl/x64/1.9/julia-1.9.3-musl-x86_64.tar.gz
+else
+  curl -o julia-1.9.3.tar.gz https://julialang-s3.julialang.org/bin/linux/x64/1.9/julia-1.9.3-linux-x86_64.tar.gz
+fi
+tar -xvf julia-1.9.3.tar.gz
+julia-1.9.3/bin/julia -e 'using InteractiveUtils; versioninfo()'
+
+# Make the host compile tools easily accessible when cross-compiling.
+# Otherwise, CMake will use the cross-compiler for host tools.
+export backup_libdir=$libdir
+
+export AR=$HOSTAR
+export AS=$HOSTAS
+export CC=$HOSTCC
+export CXX=$HOSTCXX
+export DSYMUTIL=$HOSTDSYMUTIL
+export FC=$HOSTFC
+export includedir=$host_includedir
+export libdir=$host_libdir
+export LIPO=$HOSTLIPO
+export LD=$HOSTLD
+export NM=$HOSTNM
+export OBJCOPY=$HOSTOBJCOPY
+export OBJDUMP=$HOSTOBJDUMP
+export RANLIB=$HOSTRANLIB
+export READELF=$HOSTREADELF
+export STRIP=$HOSTSTRIP
+
+# Build OR-Tools.
+cmake -S. -Bbuild \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DBUILD_DEPS:BOOL=ON \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_EXAMPLES:BOOL=OFF \
+    -DBUILD_SAMPLES:BOOL=OFF \
+    -DUSE_SCIP:BOOL=OFF \
+    -DUSE_HIGHS:BOOL=OFF \
+    -DUSE_COINOR:BOOL=OFF \
+    -DUSE_GLPK:BOOL=OFF
+cmake --build build
+cmake --build build --target install
+
+# Automatically generate the Julia bindings.
```

Comment:
I'm confused why you can't do this in post processing outside of this recipe.

Reply:
To ensure that the generated code has exactly the same version as the binary. This is just for generated code (using ProtoBuf.jl), nothing written by hand. 

Where would you do this step?